src/main/java/org/jenkinsci/plugins/ewm/definitions/Disk.java

```diff
@@ -47,15 +46,15 @@ public String getDiskId() {
 
     @CheckForNull
     public String getName() {
-        return name;
+        return name != null ? name : diskId;
     }
 
     @CheckForNull
     public String getMasterMountPoint() {
         return masterMountPoint;
     }
 
-    @CheckForNull
+    @Nonnull
     public String getPhysicalPathOnDisk() {
         return physicalPathOnDisk;
```

Comment:
This method may still return null if somebody updates from alpha-1. The best approach would be to add readResolve() method (see https://wiki.jenkins-ci.org/display/JENKINS/Hint+on+retaining+backward+compatibility). Not important for the current project stage